src/dotnet-core-uninstall/LocalizableStrings.resx

```diff
@@ -282,4 +267,31 @@
   <data name="ListX86OptionDescription" xml:space="preserve">
     <value>List x86 .NET Core SDKs or Runtimes.</value>
   </data>
+  <data name="DryRunOptionDescription" xml:space="preserve">
```

Comment:
Remember to let Kathleen review the strings if you haven't

Reply:
Yes, the strings have been run through @KathleenDollard.